src/kili/services/export/__init__.py

```diff
@@ -34,6 +34,9 @@ def export_labels(  # pylint: disable=too-many-arguments
     if kili.count_projects(project_id=project_id) == 0:
         raise NotFound(f"project ID: {project_id}")
 
+    if single_file and label_format != "raw":
+        raise ValueError(f"The label format {label_format} can not be exported in a single file.")
```

Comment:
you can move this verification deeper in the code (in `KiliFromatter`), it is the responsibility of the Exporter to tell if he can handle option combinations or not.

Reply:
done, I moved it to the BaseExporter